app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,32 @@
+class RelationshipsController < ApplicationController
+  before_action :logged_in_user
+
+  def index
+    @user = User.find_by id: params[:id] || params[:followed_id]
+    redirect_to root_path unless @user
+
+    @title = t ".#{params[:type]}"
+    @users = @user.send(params[:type]).page(params[:page]).per Settings.collections.follower_default_page
```

Comment:
why you use send method here

Reply:
because I want to calling from _stats.html.erb refer to relationships_path. Or can I use show method to represent send method?